CMakeLists.txt

```diff
@@ -414,6 +414,7 @@ if (HCC_INTEGRATE_ROCDL)
             -DLLVM_DIR=${CLANG_BIN_DIR}
             -DAMDHSACOD=${AMDHSACOD}
             -DGENERIC_IS_ZERO=ON
+            -DBUILD_HC_LIB=ON
```

Comment:
Yes, the switch was added in http://git.amd.com:8080/#/c/82189/ and is not arrived yet.
The will not brake build, however warning of unused variable will be produced.